WcaOnRails/app/views/regional_organizations/index.html.erb

```diff
@@ -1,43 +1,15 @@
-<% provide(:title, t('regional_organizations.title')) %>
-
-<div class="container">
-  <h1><%= yield(:title) %></h1>
-
-  <% if @acknowledged_regional_organizations.empty? %>
-    <p><%= t('regional_organizations.empty') %></p>
+<% img_paths = [] %>
+<% @acknowledged_regional_organizations.each do |ro| %>
+  <% if ro.logo.attached? %>
+    <% img_paths.append(polymorphic_url(ro.logo.variant(resize: "500x300"))) %>
   <% else %>
-    <p><%= t('regional_organizations.content') %></p>
-
-    <div class="organizations-list">
-      <% @acknowledged_regional_organizations.each do |ro| %>
-        <div class="organization-box">
-          <%= link_to ro.website, target: "_blank", class: "hide-new-window-icon" do %>
-            <%= image_tag ro.logo.variant(resize: "500x300") if ro.logo.attached? %>
-            <div class="organization-info <%  if ro.logo.attached? %>hide-until-hover<% end %>">
-              <div class="country">
-                <%= ro.country %>
-              </div>
-              <div class="name">
-                <%= ro.name %>
-              </div>
-            </div>
-          <% end %>
-        </div>
-      <% end %>
-    </div>
+    <% img_paths.append("") %>
   <% end %>
+<% end %>
```

Comment:
There's absolutely no need to compute this within the view.
In `regional_organizations_controller`, you can paste this exact code without all of the `<%` tags and provide it to the view by prepending `@` to the `img_paths` variable.

Reply:
:hushed: Oh I should have done that, guess it's time to refresh my Rails knowledge